ERCS/erc-7817.md

```diff
@@ -0,0 +1,112 @@
+        {
+          "name": "_spender",
+          "type": "address",
+          "values": ["0xE592427A0AEce92De3Edee1F18E0157C05861564"]
+        },
+        {
+          "name": "_value",
+          "type": "uint256"
+        }
+      ],
+      "name": "approve",
+      "payable": false,
+      "chainIds": [1],
+      "targets": ["0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"]
+    }
+  ],
+  "metadata": {
+    "commit": "70edc32"
+  }
+}
+
+```
+
+- **version**: REQUIRED. Version of the specification.
+- **report**: OPTIONAL. URL to notify in case transactions are blocked. MUST include the raw unsigned 0x-hex-encoded transaction as the `tx` URL parameter in an HTTP GET request. e.g. `https://mywebapp.xyz/report?tx=0x...`
+- **rules**: OPTIONAL. An array of rules for transaction validation. Extension of the ABI specification.
+    - **name**: OPTIONAL. Name of the function. If omitted, it MUST be interrupted as the `fallback` function.
+    - **description**: OPTIONAL. Description for front-end messages.
+    - **inputs**: OPTIONAL. Extended function ABI that defines a list of permittable `values`. It MUST be ignored if no function `name` is specified.
+        - **name:** OPTIONAL. User-readable name of the argument.
+        - **type:** REQUIRED. ABI type of argument.
+        - **values**: REQUIRED. List of valid values. It MUST be an encoded hex string of the correct byte length.
+    - **payable**: OPTIONAL. Indicates whether the function is payable. MUST default to `false` if not specified.
+    - **chainIds**: OPTIONAL. A List of chain IDs to which the rule applies. If not specified, it MUST default to any chain.
+    - **targets**: OPTIONAL. Valid "to" addresses for the transaction. MUST default to any address if not specified.
+- **metadata**: OPTIONAL. Any valid JSON to include in the document but should not be interpreted by wallets. Typically, it will include a commit hash or change control of the policy document.
+
+### Discovery
+
+It is envisioned that the policy document may be discovered and obtained using different protocols. However, the discovery protocol MUST adhere to the following:
+
+- Provide a high level of confidence that only the legitimate owner of a domain can associate a policy document to their domain.
+- Be publicly accessible.
+- Enable verification the policy document's integrity.
+
+## Rationale
+
+The front ends of dApps are regularly targeted using supply-chain attacks or DNS hijacking, whether DNS cache poisoning or registrar compromise. When executed, these attacks inject malicious JavaScript that invokes transaction signing requests to drain users' wallets. In most scenarios, this behavior is not part of the expected behavior of the dApp from which the request originates. Therefore, if dApp developers can define the intended behavior of their dApp and have wallets obtain this policy out-of-band to enforce the intended behavior or warn users, several front-end compromise risks can be mitigated.
```

Comment:
This paragraph is justifying the document as a whole, and should be moved to the motivation section.